speaker/layer2_controller.go

```diff
@@ -18,13 +18,14 @@ import (
 	"bytes"
 	"crypto/sha256"
 	"net"
+	"reflect"
 	"sort"
 
 	"github.com/go-kit/kit/log"
 	"github.com/hashicorp/memberlist"
 	"go.universe.tf/metallb/internal/config"
 	"go.universe.tf/metallb/internal/layer2"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Your editor is doing this. I had this with VS Code and was able to disable this behavior, however I don't remember what I did :-/

Reply:
yeah, some golang vim plugin(s) in my case ...